tools/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.19.2-buster
+FROM golang:1.19
```

Comment:
`golang:1.19` is not an image but everytime that there is a new release for `golang:1.19.x` it gets updated.

I think it is better to use the current latest (whatever 1.19 is pointing out, I think `1.19.4`). It would be great to be able to pin everything and just update when we intend to do that. 

Something changed without us having control over it and in this case using the latest golang would work. However, it could happen the opposite, that golang is updated and fails with an old git version for example.
It would be great if we could pin everything. If not, I think it is better one pinned than none.

Reply:
I thought about pinning the patched version, but I don't think it'll work here. Actually, using a patched version is where the bug came from. Previously we had `1.19.2`, if you check its status on docker hub you'll see it's updated two months ago. And for some unknown reason, it just didn't work starting earlier this week. So I think it's better to pin the minor version, `1.19`, and always use the latest patch.

Reply:
If I am not wrong the image was updated two months ago because it was not an update but the release (it was tagged ~ [2022-10-04](https://tip.golang.org/doc/devel/release#go1.19.minor). The `1.19` tag was updated to point to that version two months ago, but now points to `1.19.4`.

> And for some unknown reason, it just didn't work starting earlier this week

This kind of make my point. If things can change without use doing it intentionally things can break. If we point to `golang:1.19`, whenever there is a new release (for `1.19.x`) it will change without any input from us. In this case it would have fixed the CI but it can also beak it in the future.

I think we can live with it, but I am not sure if it is what we want.

Reply:
FWIW, this is just the `tools` package we use to run all the linters and stuff. We _should_ continue to pin a specific version for the builder container, which is what we use to create the release themselves.